orangecontrib/text/widgets/owcorpusviewer.py

```diff
@@ -303,11 +322,12 @@ def regenerate_docs(self):
 
     def refresh_search(self):
         if self.corpus is not None:
+            self.regexp_filter = self.filter_input.text()
```

Comment:
This is probably redundant if you refactor the gui.lineEdit to automatically sync the value.